examples/self-contained/docker-compose-build.yaml

```diff
@@ -7,6 +7,8 @@ services:
     build:
       context: ../../
       dockerfile: docker/full.Dockerfile
+    environment:
+      - ICEBERG_REST__ALLOW_ORIGIN=http://localhost:3002
```

Comment:
Why do we need it only in -build.yaml? Shouldn't it be part of the main docker-compose as well?

Ah - because UI is only here?

Reply:
Yep